python/demo/demo_static-condensation.py

```diff
@@ -42,6 +42,7 @@
 from dolfinx.io import XDMFFile
 from dolfinx.jit import ffcx_jit
 from dolfinx.mesh import locate_entities_boundary, meshtags
+from dolfinx.utils import numba_ufcx_kernel_signature as ufcx_signature
```

Comment:
I would propose we move this routine to FFCx so it sits with the C definition of the UFCx interface. I already started a branch so happy to do it?